### PR TITLE
[Build]: Fix build dependency on parameterized issue

### PR DESCRIPTION
**Pull request by xumia**
#### Why I did it
Fix the build failure as below:
___________ ERROR collecting tests/hostcfgd/hostcfgd_radius_test.py ____________
ImportError while importing test module '/sonic/src/sonic-host-services/tests/hostcfgd/hostcfgd_radius_test.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/hostcfgd/hostcfgd_radius_test.py:10: in <module>
    from parameterized import parameterized
E   ModuleNotFoundError: No module named 'parameterized'
See https://dev.azure.com/mssonic/build/_build/results?buildId=19873&view=logs&j=88ce9a53-729c-5fa9-7b6e-3d98f2488e3f&t=8d99be27-49d0-54d0-99b1-cfc0d47f0318

#### How I did it
Install the python package in sonic-slave-buster.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

### CODE REVIEW

**Comment by qiluo-msft** [comment]
How does it work before?

**Review by qiluo-msft — APPROVED** [review]

**Comment by novikauanton** [comment]
[FYI] Merged PR with the same issue https://github.com/Azure/sonic-buildimage/pull/7900.

File: sonic-slave-buster/Dockerfile.j2 [hunk 1]
```diff
@@ -421,6 +421,8 @@ RUN pip2 install nose==1.3.7
 RUN pip3 install nose==1.3.7
 RUN pip2 install mockredispy==2.9.3
 RUN pip3 install mockredispy==2.9.3
+RUN pip2 install parameterized==0.8.1
```
Comment (jleveque) [inline sonic-slave-buster/Dockerfile.j2:424]: Do we need to install the Python 2 package?

**Comment by xumia** [comment]
> How does it work before?

@qiluo-msft , it is a package version upgrade issue.
M2Crypto [required: Any, installed: 0.37.1]
    - parameterized [required: Any, installed: 0.8.1]
    
    
M2Crypto==0.38.0, it is not depended on parameterized

**Comment by jleveque** [comment]
> > How does it work before?
> 
> @qiluo-msft , it is a package version upgrade issue.
> M2Crypto [required: Any, installed: 0.37.1]
> - parameterized [required: Any, installed: 0.8.1]
> 
> M2Crypto==0.38.0, it is not depended on parameterized

Ah, so previously, when we were installing M2Crypto==0.37.1, parameterized was getting installed implicitly. But when M2Crypto upgraded to 0.38.0, parameterized was no longer getting installed, correct?

**Comment by xumia** [comment]
> Ah, so previously, when we were installing M2Crypto==0.37.1, parameterized was getting installed implicitly. But when M2Crypto upgraded to 0.38.0, parameterized was no longer getting installed, correct?
Yes, you are right.

**Comment by xumia** [comment]
@jleveque , the PR  #7900 , so this PR can be closed, right?

**Comment by jleveque** [comment]
> > Ah, so previously, when we were installing M2Crypto==0.37.1, parameterized was getting installed implicitly. But when M2Crypto upgraded to 0.38.0, parameterized was no longer getting installed, correct?

> > Yes, you are right.

Thanks! Glad to have an explanation for why this suddenly became an issue.

**Comment by jleveque** [comment]
> @jleveque , the PR #7900 merged, so this PR can be closed, right?

Yes, this PR should be unnecessary.

**Comment by jleveque** [comment]
Closing this PR in favor of https://github.com/Azure/sonic-buildimage/pull/7900

**Pull request closed without merging**